Mathlib/LinearAlgebra/Reflection.lean

```diff
@@ -174,33 +174,47 @@ lemma Dual.eq_of_preReflection_mapsTo' [CharZero R] [NoZeroSMulDivisors R M]
 variable {y}
 variable {g : Dual R M}
 
+/-- Composite of reflections in "parallel" hyperplanes is a shear (special case). -/
+lemma reflection_reflection_iterate (hfx : f x = 2) (hgy : g y = 2) (hgxfy : f y * g x = 4)
+    (n : ℕ) : ((Module.reflection hgy).trans (Module.reflection hfx))^[n] y =
+      y + (n : ℤ) • (f y • x - (2 : ℤ) • y) := by
```

Comment:
I think it will be slightly more convenient to use `(2 : R)`. I also prefer not to break place new lines in a term (the equalty in this case) unless necessary. Thus I suggest:
```suggestion
lemma reflection_reflection_iterate
    (hfx : f x = 2) (hgy : g y = 2) (hgxfy : f y * g x = 4) (n : ℕ) :
    ((reflection hgy).trans (reflection hfx))^[n] y = y + n • (f y • x - (2 : R) • y) := by
```